apps/shelter-web/src/app/app.tsx

```diff
@@ -0,0 +1,31 @@
+import { Link, Route, Routes } from 'react-router-dom';
+
+import { routeChildren } from './routes/appRoutes';
+
+export function App() {
+  return (
+    <div className="px-24">
+      <div role="navigation">
```

Comment:
@tglaz, after doing a quick lookup I think this comment makes sense.

Reply:
That's code from the template, and I just left it there so there's something to change and click for this setup test. But otherwise it'll get ripped out.